### DISCUSSION TITLE
텔레그램 질문

### DISCUSSION THREAD

**Discussion by iwantvaccine** [changdoc/naver-vaccine-macro#217] [Q&A] [answered]
봇 토큰이랑 챗 아이디에 토큰이랑 아이디 써봐도 연동이 안되네요 ㅠㅠ
무슨 정보를 넣어야하는지 알려주시면 감사하겠습니다!

**Comment by changdoc** [comment, marked as answer]
https://gabrielkim.tistory.com/entry/Telegram-Bot-Token-%EB%B0%8F-Chat-Id-%EC%96%BB%EA%B8%B0
이 곳 참고하시면 될 것 같습니다.

**Reply by iwantvaccine** [reply]
챗 아이디가 저거였군요 감사합니다!
